src/tests/map/all.mk

```diff
@@ -19,11 +19,17 @@ $(MAP_OUTPUT): $(MAP_UNIT_BIN) | $(BUILD_DIR)/tests/map/
 #
 $(BUILD_DIR)/tests/map/%.out: $(top_srcdir)/src/tests/map/%
```

Comment:
The target should be changed to ``$(BUILD_DIR)/tests/map/%.ok``.  The build product is the 'ok' files, not the log output.

Reply:
Make clean should be changed to remove the .ok files in addition to the log files.